src/status.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env browser */
+import bindMethods from "uitil/method_context";
+
+export const TAG = "simplete-status";
+const DEFAULTS = {
+	rootSelector: "simplete-form"
+};
+
+export default class SimpleteStatus extends HTMLElement {
+	// NB: `self` only required due to document-register-element polyfill
```

Comment:
OT: Do we still need to account for IE 11 in 2023?

Reply:
Here I was copying the implementation from the `SimpleteForm` and `SimpleteSuggestions` components. Answer: I am still keeping an eye on legacy browsers for a customer who still has ~25% of users using a deprecated browser.

It's not as important any more to support old browsers, but we are _also_ no longer using the `document-register-element` polyfill because it has been deprecated in favor of the `@ungap/custom-elements` polyfill. I've tested the demo with the `@ungap/custom-elements` polyfill and it works with that even when we don't pass the `self` parameter into the constructor (note that I did have to make a small tweak to the demo so that it would work with IE11).

I've separated this into this commit: 63d5de36bfb4c308fbd4710f92839ac23aa8e63b and would keep that as a separate entry in the Git history.

The other changes I have squashed into a single commit representing the addition of the `simplete-status` component.

Reply:
Excellent, thank you for that!